src/utils/image_server.cljs

```diff
@@ -1,13 +1,42 @@
 (ns utils.image-server
-  (:require [utils.datetime :as datetime]))
+  (:require
+    [status-im.utils.fs :as utils.fs]
```

Comment:
could you please move it to `react-native.fs`

Reply:
done